conftest.py

```diff
@@ -455,6 +458,57 @@ def service_actions():
         httpd.serve_forever()
 
 
+def run_in_pyodide(*args):
```

Comment:
not sure `*args` is the best options. `packages=['package1']` might be better. Also we need to be able to switch between `selenium` and  `selenium_standalone` fixtures.

In terms of API maybe something like,
```py
@run_in_pyodide(selenium, packages=['numpy')
def test_...
```

would be more explicit.


Actually I don't understand how the selenium fixture is getting in there, since it only the argument to `wrapped`, not a closure.

Reply:
What is the difference between selenium and selenium_standalone? selenium shares an instance within each file and selenium_standalone is independent? How do we decide which to use?

Reply:
Yes, all tests with the `selenium` fixture share the same session and those with `selenium_standalone` don't. Which also means they take much longer because pyodide needs to be reloaded.

Because of the performance cost of `selenium_standalone` the idea is to always `selenium` except when one needs a new sessions. For instance, it's problematic to test `loadPackage` with `selenium` because the package might have already been loaded in some earlier test. That's one of the the reason package tests take so long BTW.

Reply:
Can we reuse the browser instance with a different "tab"?

Reply:
Yeah, there are probably ways to improve it.